### ISSUE TITLE
":" breaks tridactyl

### ISSUE THREAD

**Issue by caputchinefrobles** [cmcaine/tridactyl#399]
I don't know if it's already mapped or maybe just is just something at my host but after the last release (1.9.2) every time I hit ":" for command mode I get no more response from tridactyl. 

I tried disabling every other add-on and same occurs. Nothing different at the console... 

Firefox Version: 59.0.2 (64-bit)

Didn't figure how to escape this... Tried refresh, closing tab and opening again, etc. I works closing window and starting a new one, but as soon as I hit ":" everything stops again. 

Is there a way of tagging debug mode to show more info at console logs? If so I'd be glad to give more info on the problem so it can be more of help. 

Thanks

**Comment by glacambre** [comment]
I think I've been having the same issue. I solved it by removing Tridactyl, restarting firefox and reinstalling Tridactyl. It happens every time an update is released.
I have the feeling it's linked to https://github.com/cmcaine/tridactyl/pull/326 and maybe https://github.com/cmcaine/tridactyl/pull/225 although I don't have any proof.

**Comment by cmcaine** [comment]
Try restarting your browser.

If that doesn't work you can `set logging.controller debug` and check out
the browser console.

You should see entries with fillcmdline when you press :

**Comment by caputchinefrobles** [comment]
Restart didnt work. I also tried @glacambre suggestion with no success. 

In this case, setting logging.controller in debug is not possible because I cant use command mode. Is it possible setting it other way? (possibly from storage?)

**Comment by cmcaine** [comment]
Duh.

Open the new tab page, then the web console and execute this

tri.config.set('logging', 'controller', 4)

**Comment by caputchinefrobles** [comment]
Closing this issue due to a misconfig bind I did on a very previous version of Tridactyl. 

I had a config like: "nmaps":{"J":"tabprev","K":"tabnext","**:bb**":"tabopen www.blah..."....

That ":bb" somehow was messing with the keys at cmdline input in this release. That was me trying to do what "command" does before knowing it existed. Anyway, as soon as I deleted that bind everything started working again. 

Thanks for the help.
